notificationbox/implementation.js

```diff
@@ -59,7 +59,10 @@ class Notification {
       }
     };
 
-    this.getNotificationBox().appendNotification(options.label, notificationId, imageURL, options.priority, buttons, callback);
+    let element = this.getNotificationBox().appendNotification(options.label, notificationId, imageURL, options.priority, buttons, callback);
+    for (let key in options.style) {
```

Comment:
John and I discussed to make a whitelist here. John said he already has the code in mind and wants to add it himself. Thanks, John!

Reply:
Allowed should be things like `background`*, `font`*, `color`, `margin`*, `padding`* etc.